homeassistant/components/hue/binary_sensor.py

```diff
@@ -43,10 +44,17 @@ def device_state_attributes(self):
 
 SENSOR_CONFIG_MAP.update(
     {
-        TYPE_ZLL_PRESENCE: {
-            "platform": "binary_sensor",
-            "name_format": PRESENCE_NAME_FORMAT,
-            "class": HuePresence,
-        }
+        TYPE_ZLL_PRESENCE: [
+            {
+                "platform": "binary_sensor",
+                "name_format": PRESENCE_NAME_FORMAT,
+                "class": HuePresence,
+            },
+            {
+                "platform": "sensor",
+                "name_format": REMOTE_NAME_FORMAT,
+                "class": HueBattery,
```

Comment:
The sensor entity class `HueBattery` is not compatible with our binary sensor base. We need to adapt it as a binary sensor if we want a binary sensor for battery level.